src/components/calculation/calculator.js

```diff
@@ -0,0 +1,7 @@
+export default function Calculator(splitedNum) {
+    let sum = 0;
+    splitedNum.forEach((n) => {
+        sum += n;
+    });
+    return sum;
```

Comment:
reduce를 쓰면 sum 변수를 선언하지 않아도 되어서 SNXWXH님 말씀처럼 깔끔할 것 같습니다! 🤔
```suggestion
   return splitedNum.reduce((a,b)=>a+b,0)
```